config/locales/en.yml

```diff
@@ -724,6 +724,7 @@ en:
     already_ordered:
       cart: "cart"
       message_html: "You have an order for this order cycle already. Check the %{cart} to see the items you ordered before. You can also cancel items as long as the order cycle is open."
+    items_removed_from_oc: "Items in your cart have been removed from the order cycle. Please review your order before proceeding."
```

Comment:
I'd like not to use the `oc` abbreviation in the codebase. It adds friction for new contributors and it makes it harder for current devs to understand it. I don't think it's a big deal having a longer name if it helps comprehension.